app/uk/gov/hmrc/transitmovements/repositories/DeparturesRepository.scala

```diff
@@ -70,7 +72,7 @@ class DeparturesRepositoryImpl @Inject() (
             result => Right(createResponse(result))
           )
       case Failure(ex) =>
-        Future.successful(Left(ex))
+        Future.successful(Left(OtherError(Some(ex))))
```

Comment:
Other error seems a bit unserious here, maybe UnexpectedError? Since we've not got something explicit for it?